# 单个技能的信息
string skill         # 技能名称
Parameter[] params   # 参数列表，目前可扩展为任意键值对
string object_id     # 关联对象 ID，若无则留空
string task_id       # 所属任务 ID
